salt/metalk8s/addons/monitoring/grafana/deployed.sls

```diff
@@ -7476,6 +7489,13 @@ spec:
       - configMap:
           name: grafana-dashboard-statefulset
         name: grafana-dashboard-statefulset
+      tolerations:
```

Comment:
Is it possible to also specify a `nodeSelector` for `node-role.kubernetes.io/infra: ''`? This also applies for all other 'infra' services you're deploying here.

Reply:
Tracked in #1169.